tools/ktx/command_create.cpp

```diff
@@ -1802,6 +1823,15 @@ void CommandCreate::executeCreate() {
             if (options.swizzleInput)
                 image->swizzle(*options.swizzleInput);
 
+            if (options.premultiplyAlpha) {
+                if(image->getComponentCount() < 4) {
+                    // This branch is unreachable because all images from loadInputImage already contain alpha channel.
```

Comment:
I think you mean "none of the images contain an alpha channel".  But I don't think the comment adds anything you can't tell from reading the error message. I suggest deleting it.